### PR TITLE
[Fix]Fix metafile

### PR DESCRIPTION
**Pull request by HIT-cwh**
Hi! This pr fixed the metafile.yml in configs in mmcls. Pls take a look.

### CODE REVIEW

File: configs/fp16/metafile.yml [hunk 1]
```diff
@@ -1,30 +1,30 @@
 Collections:
   - Name: FP16
     Metadata:
-      Training Data: ImageNet
-      Training Resources: 8x V100 GPUs
+      Training Data: ImageNet-1k
```
Comment (zhouzaida) [inline configs/fp16/metafile.yml:4]: Is it necessary to replace ImageNet with ImageNet-1k

Comment (HIT-cwh) [reply configs/fp16/metafile.yml:4]: Fixed. thx

Comment (HIT-cwh) [reply configs/fp16/metafile.yml:4]: Maybe not, but I just fellow the example in PWC Model-index Format.

Comment (zhouzaida) [reply configs/fp16/metafile.yml:4]: @ZwwWayne please have a look

**Review by ycxioooong — APPROVED** [review]
LGTM

**Pull request merged**